src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -435,6 +435,9 @@ int IDebugEngineLaunch2.LaunchSuspended(string pszServer, IDebugPort2 port, stri
                     {
                         try
                         {
+                            // Check if the logger was enabled late.
```

Comment:
nit: might want to do this at the beginning of 'LaunchSuspended' since it should be set by then, and if we wind up adding any logging, it would be good ta capture it.